Source/CarthageKit/Git.swift

```diff
@@ -18,11 +18,16 @@ public struct FetchCache {
 
 	private static var lastFetchTimes: [GitURL: TimeInterval] = [:]
 
+	private static let lock = NSLock()
+
 	internal static func clearFetchTimes() {
 		lastFetchTimes.removeAll()
```

Comment:
If the lock is guarding usage of `lastFetchTimes`, do we need the lock here as well?

Reply:
As you pointed out, it needed to be added!